ql/src/test/queries/clientpositive/insert_overwrite_drop_partition.q

```diff
@@ -0,0 +1,46 @@
+set hive.exec.dynamic.partition.mode=nonstrict;
+set hive.stats.column.autogather=false;
```

Comment:
are these configs (from line 2 to 4) needed to be set for this scenario ?

Reply:
No, removed the configs as default value of hive.exec.dynamic.partition.mode is nonstrict